bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/transactions/TransactionResourceManager.java

```diff
@@ -96,6 +118,51 @@ public static TransactionResourceManager getInstance() {
         return transactionResourceManager;
     }
 
+    /**
+     * This method sets values for atomikos transaction log path and name properties using the available configs.
+     *
+     */
+    private void setLogProperties() {
+        final Path projectRoot = findProjectRoot(Paths.get(System.getProperty("user.dir")));
+        if (projectRoot != null) {
+            String logDir = getTransactionLogDirectory();
+            String logPath = projectRoot.toAbsolutePath().toString() + "/" + logDir;
+            Path transactionLogDirectory = Paths.get(logPath);
+            if (!Files.exists(transactionLogDirectory)) {
+                try {
+                    Files.createDirectory(transactionLogDirectory);
+                } catch (IOException e) {
+                    stderr.println("error: failed to create transaction log directory");
+                }
+            }
+            System.setProperty(ATOMIKOS_LOG_BASE_PROPERTY, logPath);
+            System.setProperty(ATOMIKOS_LOG_NAME_PROPERTY, "transaction_recovery");
+        }
+    }
+
+    /**
+     * This method checks whether the atomikos transaction manager should be enabled or not.
+     *
+     * @return boolean whether the atomikos transaction manager should be enabled or not
+     */
+    private boolean getTransactionManagerEnabled() {
+        boolean transactionManagerEnabled = CONFIG_REGISTRY.getAsBoolean("b7a.transaction.manager.enabled");
```

Comment:
Let's move b7a.transaction.manager.enabled to a constant

Reply:
Fixed in 91ca624